app/pages/home-not-logged-in.jsx

```diff
@@ -31,7 +31,7 @@ export default class HomePage extends React.Component {
       featuredProject: null,
       promotedProjects: [],
       screenWidth: 0,
-      volunteerCount: 1500000
+      volunteerCount: 0
```

Comment:
So this now means the counter starts at 0 on the home page until the API request for users completes?

Reply:
I think that may be true.  But as we found out during the classification count work (see #4526), this initialization value was being added to the other constant values -and- the returned value, so I mimicked that fix to be sure such an offset didn't occur here.  Not exactly sure how/when the update from initialized value versus API requested value occurs, but know this is the safe route.